config.yaml

```diff
@@ -22,3 +22,33 @@ options:
       Supported units: y, w, d, h, m, s.
     type: string
     default: "1m"
+  always_enable_zipkin:
+    description: > 
+      Force-enable the receiver for the 'zipkin' protocol in Grafana Agent, 
+      even if there is no integration currently requesting it.
+    type: boolean
+    default: false
+  always_enable_otlp_grpc:
+    description: > 
+      Force-enable the receiver for the 'otlp_grpc' protocol in Grafana Agent, 
+      even if there is no integration currently requesting it.
+    type: boolean
+    default: false
+  always_enable_otlp_http:
+    description: > 
+      Force-enable the receiver for the 'otlp_http' protocol in Grafana Agent, 
+      even if there is no integration currently requesting it.
+    type: boolean
+    default: false
+  always_enable_jaeger_grpc:
+    description: > 
+      Force-enable the receiver for the 'jaeger_grpc' protocol in Grafana Agent, 
+      even if there is no integration currently requesting it.
+    type: boolean
+    default: false
+  always_enable_jaeger_thrift_http:
+    description: >
+      Force-enable the receiver for the 'jaeger_thrift_http' protocol in Grafana Agent, 
+      even if there is no integration currently requesting it.
+    type: boolean
+    default: false
```

Comment:
- When would we want to enable receivers without an integration?
- I imagine typical use cases would have only one kind. If so, should we merge all these into a string type? Comma separated?

Reply:
1): when sending traces from an uncharmed application
2): I guess that's true, given we don't know anything about the typical use case yet :) as for the 'single-option-but-comma-separated' vs 'many-options' discussion, I thought the general conclusion was that the many-options is to be preferred. Same arguments apply as the 'roles' configuration in mimir, loki, tempo.

Reply:
The behaviour also matches this ADR: https://github.com/canonical/observability/blob/main/decision-records/2024-06-25--extra-tempo-receivers.md 

I can imagine a scenario where grafana-agent could have more than 1 receiver type force-enabled, in case there'd be more than 1 uncharmed application that would be using it. So I think it should stay that way.